### PR TITLE
Add Treact templates and components for React

### PR DESCRIPTION
**Pull request by MohanedAshraf**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -107,6 +107,7 @@
 - [a17t](https://a17t.miles.land) - Atomic design toolkit built to extend Tailwind CSS.
 - [tailblocks](https://mertjf.github.io/tailblocks) - 60+ different ready to use Tailwind CSS blocks.
 - [Meraki UI Components](https://merakiui.com) - Beautiful Tailwind CSS components that support RTL languages.
+- [Treact](https://treact.owaiskhan.me) - React UI Templates and Components built using Tailwind CSS.
```
Comment (innocenzi) [inline readme.md:110]:
```suggestion
+ [Treact](https://treact.owaiskhan.me) - React UI templates and components built using Tailwind CSS.
```

Comment (innocenzi) [reply readme.md:110]: Looks good, can you change the casing here though?

**Comment by joshmanders** [comment]
These are amazing @MohanedAshraf!

**Comment by MohanedAshraf** [comment]
> These are amazing @MohanedAshraf!

Actually these aren't my work. I coincidentally found it and it helped me a lot and I want people to get to know about it.

**Pull request merged**